ivy/container/extensions.py

```diff
@@ -1370,3 +1370,125 @@ def fmax(
         }
         """
         return self.static_fmax(self, x2, out=out)
+
+
+    @staticmethod
+    def static_nansum(
+        input: ivy.Container,
+        /,
+        *,
+        axis: Optional[Union[Tuple[int], int]] = None,
+        dtype: Optional[Union[ivy.Dtype, ivy.NativeDtype]] = None,
+        keepdims: Optional[bool] = False,
+        key_chains: Optional[Union[List[str], Dict[str, str]]] = None,
+        to_apply: bool = True,
+        prune_unapplied: bool = False,
+        map_sequences: bool = False,
+        out: Optional[ivy.Array] = None,
+    ) -> ivy.Container:
+        """
+        ivy.Container static method variant of ivy.nansum. This method simply wraps
+        the function, and so the docstring for ivy.nansum also applies to this method
+        with minimal changes.
+        Parameters
+        ----------
+        input
+            Input array.
+        axis
+            Axis or axes along which the sum is computed.
+            The default is to compute the sum of the flattened array.
+        dtype
+            The type of the returned array and of the accumulator in
+            which the elements are summed. By default, the dtype of input is used.
+        keepdims
+            If this is set to True, the axes which are reduced are left
+            in the result as dimensions with size one.
+        out
+            Alternate output array in which to place the result.
+            The default is None.
+        Returns
+        -------
+        ret
+            A new array holding the result is returned unless out is specified,
+            in which it is returned.
+        Examples
+        --------
+        With one :class:`ivy.Container` input:
+        >>> x = ivy.Container(a=ivy.array([[10, 7, 4], [3, 2, 1]]),\
+                b=ivy.array([[1, 4, 2], [ivy.nan, ivy.nan, 0]]))
+        >>> x.nansum(axis=0)
```

Comment:
Statis methods are not callable as an instance. Rather they are called on the class itself.  In this case, it would be `ivy.Container.static_nansum(x)`